source/installation/install-on-an-unattended-server/debian-server-installation/install-debian-server.txt

```diff
@@ -39,9 +39,9 @@ Steps
 
    a. To download and install MySQL or Oracle drivers, see the following:
 
-      - `MySQL: latest 8.0.x version, Platform Independent Version <https://dev.mysql.com/downloads/connector/j/>`_
+      - `MySQL: latest 8.3.x version, Platform Independent Version <https://dev.mysql.com/downloads/connector/j/>`_
 
-      - `Oracle: 21.6.0.0 of odjbc11.jar from the Oracle 21c <https://www.oracle.com/database/technologies/appdev/jdbc-downloads.html>`_
+      - `Oracle: 21.6.0.0 of ojdbc11.jar from the Oracle 21c <https://www.oracle.com/database/technologies/appdev/jdbc-downloads.html>`_
```

Comment:
[For reviewer]
This was an intentional update that was missed in a different ticket.